packages/runner/injection/patches/fetch.ts

```diff
@@ -1,16 +1,16 @@
-import { captureFullRequestUrl } from './utils'
+import { captureFullRequestUrl, requestSentWithCredentials } from './utils'
 
-export const patchFetch = (Cypress: Cypress.Cypress, window) => {
+export const patchFetch = (window) => {
   // if fetch is available in the browser, or is polyfilled by whatwg fetch
   // intercept method calls and add cypress headers to determine cookie applications in the proxy
   // for simulated top. @see https://github.github.io/fetch/ for default options
-  if (!Cypress.config('experimentalSessionAndOrigin') || !window.fetch) {
+  if (!window.fetch) {
     return
   }
 
   const originalFetch = window.fetch
 
-  window.fetch = function (...args) {
+  window.fetch = async function (...args) {
```

Comment:
we're now async so we can await the backend call.. i think previously it was a race condition.